ci-operator/step-registry/single-node/conf/aws/single-node-conf-aws-commands.sh

```diff
@@ -4,7 +4,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SINGLE_NODE_AWS_INSTANCE_TYPE="m5d.2xlarge"
+if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    SINGLE_NODE_AWS_INSTANCE_TYPE=${COMPUTE_NODE_TYPE}
+else
+    SINGLE_NODE_AWS_INSTANCE_TYPE="m5d.2xlarge"
+fi
```

Comment:
This seems inconsistent.  Does the x86 invocation export `COMPUTE_NODE_TYPE`?  If so, maybe just accept the variable unconditionally?  Otherwise, just hardcode `m6g[d].2xlarge` here for ARM.